src/zenml/zen_stores/schemas/artifact_schemas.py

```diff
@@ -73,7 +74,7 @@ class ArtifactSchema(NamedSchema, table=True):
     workspace: "WorkspaceSchema" = Relationship(back_populates="artifacts")
 
     type: ArtifactType
-    uri: str
+    uri: str = Field(sa_column=Column(TEXT, nullable=False))
```

Comment:
Was this needed to accommodate longer URIs ?

Reply:
Yeah one of the CI integration tests failed because of this before